python/test/pipeline_test.py

```diff
@@ -450,10 +451,10 @@ def testStartCountdown(self):
   def testStartEta(self):
     """Tests starting a pipeline with an eta."""
     stage = NothingPipeline('one', 'two', three='red', four=1234)
-    eta = datetime.datetime.utcnow() + datetime.timedelta(seconds=30)
+    eta = datetime.datetime.utcnow().replace(tzinfo=pytz.UTC) + datetime.timedelta(seconds=30)
```

Comment:
OK, I'll try that. I'll get rid of `pytz` (which for some reason I thought it was standard) and do what the thread suggests.